src/main/java/org/folio/circulation/resources/RequestNoticeSender.java

```diff
@@ -120,10 +120,13 @@ public Result<RequestAndRelatedRecords> sendNoticeOnRequestCancelled(
     log.debug("sendNoticeOnRequestCancelled:: parameters records: {}", () -> records);
     Request request = records.getRequest();
 
-    if (request.hasItemId()) {
-      sendCancellationNoticeForRequestWithItemId(request);
-    } else {
-      sendCancellationNoticeForRequestWithoutItemId(request);
+    // Send the cancellation notice only if the suppressNotification flag is false
```

Comment:
Update the comment or remove the comment because suppressNotification flag is not present anymore.

Reply:
Please add test cases in this PR

Reply:
done